pkg/patch/cmd_test.go

```diff
@@ -0,0 +1,37 @@
+package patch
```

Comment:
Could you take a pass through all of the new files and make sure that the licensing header is added?

Reply:
added